serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -1 +1,89 @@
-//= Consuming and producing events with Kafka
\ No newline at end of file
+Consuming and producing events with Kafka
+=========================================
+:compat-mode!:
+// Metadata:
+:description: Consuming and producing events with Kafka
+:keywords: kogito, workflow, serverless, events, kafka
+// links
+:kogito_messaging_url: https://docs.jboss.org/kogito/release/latest/html_single/#_kogito_messaging_add_on
+:kogito_kafka_integration_url: https://docs.jboss.org/kogito/release/latest/html_single/#proc-messaging-enabling_kogito-configuring
+:configuration_example_url: {kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/resources/application.properties}
+:smallrye_connector_doc_url: {smallrye_url}/connectors/connectors.html
+:kafka_connector_properties_url: {smallrye_url}/kafka/kafka.html#_configuration_reference
+
+
+This guide focus on how to configure Kogito Serverless Workflow to interact with link: link:{kafka_doc_url}[Kafka]. Kafka uses an abstraction called topic to classify events. Events are published to a topic and events are consumed from a topic. 
+
+Kogito uses link:{smallrye_connector_doc_url}[Smallrye connectors] to encapsulate access to several brokers. This allows Kogito to support different brokers just by changing configuration and classpath dependencies. However, it also introduces an extra layer of indirection, through an entity called channel. Channels are unidirectional, they have to be declared as input (incoming) or output (outgoing), and this definition is immutable. When using Kafka, Smallrye channels must be mapped to Kafka topics though configuration.
+
+== Setting up Kafka dependency
```

Comment:
```suggestion
== Configuring Kafka dependency
```

Reply:
I do not like configuring, because it is broad and we are also configuring the dependency, somehow, in the next section. In this section we are just adding the dependency, therefore probably adding is the right word to be used